operator/webhook/addons/certmanager/README.md

```diff
@@ -0,0 +1,15 @@
+# Certmanager Addon
+
+The Certmanager Addon creates certificates for `IntegrationRoute`s based off of annotations in the `IntegrationRoute`.
+
+See the example `IntegrationRoute` in the [README](../../../example/README.md#example-integrationroute-using-tls-and-the-certmanager-addon).
+
+## Supported Annotations

```

Comment:
We should consider supporting the `cert-manager.io/issuer` annotation as well, since we might not always deal with a cluster-level issuer.  However, this PR has already grown considerably, so I would leave it as a follow-on.

Reply:
Added